### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by kotekjedi**
Hi Ori!
I applied small changes to the dataloader, so now you can run the task with the current API.
Also, I restored the default path to imagenet (let's not commit our local paths) and replaced the local data-generation path with the global one.

Other changes are more cosmetic. Please have a look :)

* minor changes proposed to ccc_utils.py and ccc_lmdb.py
* default values restored in imagenet.py
* task's files moved to their own folder
* ccc.py reformatted and black linter applied

### CODE REVIEW

File: shifthappens/tasks/ccc/ccc.py [hunk 1]
```diff
@@ -7,80 +7,91 @@
...
 
 
 if __name__ == "__main__":
-    # from shifthappens.models.torchvision import ResNet50
-    # sh_benchmark.evaluate_model(ResNet50(), "data")
-    a = CCC('./')
-    a._prepare_dataloader()
+    from shifthappens.models.torchvision import ResNet50
+
+    sh_benchmark.evaluate_model(ResNet50(), "../../../../../test_data")
```
Comment (kotekjedi) [inline shifthappens/tasks/ccc/ccc.py]:
```suggestion
```

File: shifthappens/tasks/ccc/ccc.py [hunk 2]
```diff
@@ -7,80 +7,88 @@
...
 
-        accuracy = all_predicted_labels == np.array(self.ch_dataset.targets)
+        accuracy = (all_predicted_labels == np.array(self.targets)).mean()
 
         return TaskResult(
             accuracy=accuracy, summary_metrics={Metric.Robustness: "accuracy"}
         )
 
 
 if __name__ == "__main__":
```
Comment (kotekjedi) [inline shifthappens/tasks/ccc/ccc.py]:
```suggestion
```

File: shifthappens/tasks/ccc/ccc_utils.py [hunk 3]
```diff
@@ -266,15 +269,22 @@ def generate_dataset(self):
                 generated_subset = ApplyTransforms(self.data_dir, n1, n2, s1, s2, self.subset_size, self.target_dir)
                 dset2lmdb(generated_subset, path)
 
+            test_transform = tv_transforms.Compose(
+                [
+                    tv_transforms.ToTensor(),
+                    tv_transforms.Lambda(lambda x: x.permute(1, 2, 0)),
```
Comment (oripress) [inline shifthappens/tasks/ccc/ccc_utils.py:275]: Why is this needed?

Comment (kotekjedi) [reply shifthappens/tasks/ccc/ccc_utils.py:275]: Here we just put channels into the last dimension.  Later we undone it in the model's part [transpose](https://github.com/shift-happens-benchmark/icml-2022/blob/main/shifthappens/models/torchvision.py#L26).

Tbh I don't remember why we are doing it this way :/. But here, we need it to have the correct format later in the model's API.

Comment (oripress) [reply shifthappens/tasks/ccc/ccc_utils.py:275]: Great, thanks
